### PR TITLE
Library V1.1

### PR DESCRIPTION
**Pull request by AbdumurodovaZulfizar**
JavaScript&nbsp;<br>
### Library
It is a simple webpage to create your own library for books!
_Languages and tools:_

- JS
- Bootstrap
- HTML

_Aims of doing this project:_

- Review JS basics
- Use constructor functions
- Use JS build-in functions

by Tanzila and Zulfizar

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+        <label for="exampleInputEmail1" class="form-label">Book Name</label>
+        <input type="text" class="form-control" id="bookName" aria-describedby="emailHelp">
+      </div>
+      <div class="mb-3">
+        <label for="exampleInputPassword1" class="form-label">Author</label>
+        <input type="text" class="form-control" id="author">
+      </div>
+      <div class="mb-3">
+        <label for="exampleInputPassword1" class="form-label">Pages</label>
+        <input type="integer" class="form-control" id="pages">
```
Comment (addod19) [inline index.html]: Kindly change the type to "number" so the input can accept only numbers

File: script.js [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+const form = document.querySelector('#form');
+const read = document.getElementById('readBookCheck');
+const display = document.querySelector('#display');
+
+button.addEventListener('click', (e) => {
+  e.preventDefault();
+  document.querySelector('#form').classList.remove('d-none');
+});
+
+/* eslint-disable no-use-before-define */
```
Comment (addod19) [inline script.js]: Kindly remove the inline ESLint disable rule

File: script.js [hunk 3]
```diff
@@ -0,0 +1,102 @@
...
+function updateBookList() {
+  display.innerHTML = '';
+  bookArray.forEach((book, index) => createNewBook(book, index));
+}
+/* eslint-enable no-use-before-define */
```
Comment (addod19) [inline script.js]: Kindly remove the inline ESLint disable rule

File: script.js [hunk 4]
```diff
@@ -0,0 +1,102 @@
...
+  cardBody.appendChild(deletebutton);
+
+  AllBooks.appendChild(cardBody);
+  display.appendChild(AllBooks);
+
+  endbutton.addEventListener('click', EndBook);
+  deletebutton.addEventListener('click', DeleteBook);
+}
+
+function addBookToLibrary(e) {
```
Comment (addod19) [inline script.js]: [Optional]
- Let us use ES6 arrow syntax instead..... ``` const addBookToLibrary = e => { .... } ```

**Review by addod19 — CHANGES_REQUESTED** [review]
## Status: Require changes :recycle: :traffic_light: 

Hi @AbdumurodovaZulfizar and @tanzila-abedin, congratulations on this great project :1st_place_medal: The README is professional :+1: The PR and commit messages easier to understand :smile: Worked with Git Flow :+1: Great DOM manipulation
However, there are a few comments we need to address before proceeding, kindly check the comments below:
- Do not use Window.alert() or Window.confirm(). It’s ugly, impossible to style, stop code execution and display differently on different browsers. Use custom modal instead
- Kindly address the inline comments below

Once these are fixed, kindly submit another code review request :smile: 
_Happy coding_ :man_technologist: 
[Daniel Larbi Addo](https://www.linkedin.com/in/daniel-larbi-addo/)

**Comment by tanzila-abedin** [comment]
Hi @addod19 

Thank you for your review.

I have made all the changes but I cannot seem to get rid of one Estlint error. 
When we place the function after it's defined then it shows the same error "used before defined" for another function. And it carries on repeating. Hence we have used  
/* eslint-enable no-use-before-define */
/* eslint-disable no-use-before-define */
around a function to stop that from recurring.

Screenshot from 2021-07-20 19-41-36

**Review by SajjadAhmad14 — CHANGES_REQUESTED** [review]
# Changes Required :recycle:
> "Change is inevitable. Change is constant. -Benjamin Disraeli"
## Hello, thank you for working on changes suggested previously :ok_hand:
## To Highlight 🎉
- [ ] You have mastered Git flow.
- [ ] README is professional.
## To Improve 🙏
- [ ] PR is showing linter errors. KIndly fix that.
# Optional


### Happy Coding ☺️

## If you have any questions you can comment here in this PR or can always connect with me on slack ( @Sajjad Ahmad ).

**Comment by tanzila-abedin** [comment]
Hi @addod19 

Thank you for your review.

I have made all the changes but I cannot seem to get rid of one Estlint error. 
When we place the function after it's defined then it shows the same error "used before defined" for another function. And it carries on repeating. Hence we have used  
/* eslint-enable no-use-before-define */
/* eslint-disable no-use-before-define */
around a function to stop that from recurring.

Screenshot from 2021-07-20 19-41-36

**Review by BrianSammit — APPROVED** [review]
STATUS: APPROVED 🟢 👏 
-----------
Hello @tanzila-abedin @AbdumurodovaZulfizar 👋, great job working on the required changes 👏 💪 
Your project looks really nice and works correctly 😄 💯 
-----------------
Now you can merge this branch 🥇 
HAPPY CODING!! AND GOOD LUCK WITH THE COMMING ONES 🤞 🍀 

kvUU7

**Pull request merged**